UnitsNet/CustomCode/Quantities/Length.extra.cs

```diff
@@ -87,25 +87,19 @@ public static bool TryParseFeetInches(string? str, out Length result, IFormatPro
             string footRegex = quantityParser.CreateRegexPatternForUnit(LengthUnit.Foot, formatProvider, matchEntireString: false);
             string inchRegex = quantityParser.CreateRegexPatternForUnit(LengthUnit.Inch, formatProvider, matchEntireString: false);
 
-            // Match entire string exactly
-            string pattern = $@"^(?<negativeSign>\-?)(?<feet>{footRegex})\s?(?<inches>{inchRegex})$";
+            var feetMatch = new Regex(footRegex, RegexOptions.Singleline).Match(str);
+            var inchesMatch = new Regex(inchRegex, RegexOptions.Singleline).Match(str);
```

Comment:
Won't this allow invalid formats like inches first, feet after? `5" 1'`
Or is this actually an acceptable form?

Reply:
You're right, it wouldn't. I'll make sure to enforce ordering.